README.md

```diff
@@ -1 +1,182 @@
-# Purple-Coders---HTML-CSS-JavaScript-capstone-project
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+ 
+  <img src="./images/purpleLogoBg.png" alt="logo" width="70%"  height="400px"/>
+  <br/>
```

Comment:
- [ ] Kindly remove all comments from your readme file, this is just a guide on how to fill in the sections of the readme.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: